src/lib/output/themes/default/DefaultTheme.tsx

```diff
@@ -167,8 +167,7 @@ export class DefaultTheme extends Theme {
             event.project.groups.forEach(DefaultTheme.applyGroupClasses);
         }
 
-        for (const id in event.project.reflections) {
-            const reflection = event.project.reflections[id];
+        for (const reflection of Object.values(event.project)) {
```

Comment:
Behavior change here! I think you meant `of Object.values(event.project.reflections)`

Reply:
Ohh, great catch!

```suggestion
        for (const reflection of Object.values(event.project.reflections)) {
```